include/AcrobatAccess/AcrobatAccess.idl

```diff
@@ -40,9 +40,15 @@ import "oleacc.idl";		//ajc - get the real definition of IAccessible from Micros
 	helpstring("IAccID Interface"),
 	pointer_default(unique)
 ]
+
+
    interface IAccID : IUnknown
 {
-	[id(22), helpstring("method get_accID")] HRESULT get_accID([out, retval] long *pID);
+#if _WIN64
+	[id(22), helpstring("method get_accID")] HRESULT get_accID([out, retval] long long*pID);

```

Comment:
Added missing space

```suggestion
	[id(22), helpstring("method get_accID")] HRESULT get_accID([out, retval] long long* pID);
```

Reply:
I would not want to make any change to the idl file (no matter how small) as this is an official file from the Adobe Acrobat SDK. (I.e. not our code).